ledger/src/blockstore_db.rs

```diff
@@ -1542,19 +1559,6 @@ where
         }))
     }
 
-    pub fn compact_range(&self, from: Slot, to: Slot) -> Result<bool>
-    where
```

Comment:
Kind of unrelated to rip this out, but guessing you just didn't feel the need to update the unused function.

There is one potential use case I could see for this (compacting warehouse blockstore before google bucket upload), but we aren't doing that currently and adding these 10 lines back in if/when we do that should be easy enough so I'm cool with ripping this out